iteration/iteration.go

```diff
@@ -127,7 +131,7 @@ func (m *GormIterationRepository) List(ctx context.Context, spaceID uuid.UUID) (
 	return objs, nil
 }
 
-// Get the Root Iteration for a space
+// RootIteration returns the Root Iteration for a space
 func (m *GormIterationRepository) RootIteration(ctx context.Context, spaceID uuid.UUID) (*Iteration, error) {
```

Comment:
@sbose78  : hahahaha , good catch. I was about to make this change.
I liked `Root` than `RootArea` as in your PR as well. so I will update the method name.